src/fireflyClient.ts

```diff
@@ -198,7 +204,7 @@ export class FireflyClient {
       .connect(this.getWallet())
       .balanceOf(this.getPublicAddress());
 
-    return bnToString(+balance);
+    return bnToString(+balance); 
```

Comment:
use `balance.toHexString()` instead of `+balance` that is the correct way of converting bignumbers returned from contracts.